### PR TITLE
added statusCode and pattern to FilterMonitorFunc

### PR DESCRIPTION
**Pull request by jerson**
add statusCode and pattern for custom monitor tools like prometheus

### CODE REVIEW

**Comment by jerson** [comment]
pleaseee

**Comment by jerson** [comment]
:(

File: admin.go [hunk 1]
```diff
@@ -37,7 +37,7 @@ var beeAdminApp *adminApp
 // FilterMonitorFunc is default monitor filter when admin module is enable.
 // if this func returns, admin module records qbs for this request by condition of this function logic.
 // usage:
-// 	func MyFilterMonitor(method, requestPath string, t time.Duration) bool {
+// 	func MyFilterMonitor(method, requestPath string, t time.Duration, statusCode int) bool {
```
Comment (astaxie) [inline admin.go]: Could you also update this example? it miss the pattern

Comment (jerson) [inline admin.go]: updated :)

**Pull request merged**